libs/wingsdk/src/cloud/queue.ts

```diff
@@ -125,7 +146,7 @@ export interface IQueueSetConsumerHandlerClient {
    * Function that will be called when a message is received from the queue.
    * @inflight
    */
-  handle(message: string): Promise<void>;
+  handle(message: string): Promise<any>;
```

Comment:
why?

Reply:
It feels like this should be an implementation detail...

@marciocadev can you try swapping the interfaces here and see if that fixes the issue? See my suggested change:

```diff
diff --git a/libs/wingsdk/src/cloud/queue.ts b/libs/wingsdk/src/cloud/queue.ts
index 3159b6748..be0f36de9 100644
--- a/libs/wingsdk/src/cloud/queue.ts
+++ b/libs/wingsdk/src/cloud/queue.ts
@@ -152,7 +152,7 @@ export interface IQueueSetConsumerHandlerClient {
    * Function that will be called when a message is received from the queue.
    * @inflight
    */
-  handle(message: string): Promise<any>;
+  handle(message: string): Promise<void>;
 }

 /**
diff --git a/libs/wingsdk/src/shared-aws/queue.setconsumer.inflight.ts b/libs/wingsdk/src/shared-aws/queue.setconsumer.inflight.ts
index b9b35caae..c63d9fa6f 100644
--- a/libs/wingsdk/src/shared-aws/queue.setconsumer.inflight.ts
+++ b/libs/wingsdk/src/shared-aws/queue.setconsumer.inflight.ts
@@ -3,11 +3,9 @@ import type {
   IQueueSetConsumerHandlerClient,
 } from "../cloud";

-export class QueueSetConsumerHandlerClient
-  implements IQueueSetConsumerHandlerClient
-{
-  private readonly handler: IFunctionHandlerClient;
-  constructor({ handler }: { handler: IFunctionHandlerClient }) {
+export class QueueSetConsumerHandlerClient implements IFunctionHandlerClient {
+  private readonly handler: IQueueSetConsumerHandlerClient;
+  constructor({ handler }: { handler: IQueueSetConsumerHandlerClient }) {
     this.handler = handler;
   }
   public async handle(event: any): Promise<any> {
```

Reply:
Maybe we can avoid swapping interfaces and instead fix this by overriding the original method when implementing the class